<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>499</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
                          background-color: #4C9F70; 
                          color: white; 
                          font-size: 16px; 
                          border: 4px solid #2C6B43; 
                          border-radius: 5px; 
                          padding: 10px 10px;
                          text-align: center;
                          }
QPushButton:pressed {
                          background-color: #3E7F58;
}
QPushButton:hover {
                          background-color: #3E7F58;
}</string>
   </property>
   <property name="text">
    <string>Create new game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
                          background-color: #4C9F70; 
                          color: white; 
                          font-size: 16px; 
                          border: 4px solid #2C6B43; 
                          border-radius: 5px; 
                          padding: 10px 10px;
                          text-align: center;
                          }
QPushButton:pressed {
                          background-color: #3E7F58;
}
QPushButton:hover {
                          background-color: #3E7F58;
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="JoinButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
                          background-color: #4C9F70; 
                          color: white; 
                          font-size: 16px; 
                          border: 4px solid #2C6B43; 
                          border-radius: 5px; 
                          padding: 5px 5px;
                          text-align: center;
                          }
QPushButton:pressed {
                          background-color: #3E7F58;
}
QPushButton:hover {
                          background-color: #3E7F58;
}</string>
   </property>
   <property name="text">
    <string>Join room</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
